sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -1190,9 +1190,10 @@ public virtual Response DecommissionModel(string modelId, CancellationToken canc
         /// Console.WriteLine($&quot;Created models &apos;{componentModelId}&apos; and &apos;{sampleModelId}&apos;.&quot;);
         /// </code>
         /// </example>
-        public virtual Task<Response<IReadOnlyList<ModelData>>> CreateModelsAsync(IEnumerable<string> models, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<ModelData[]>> CreateModelsAsync(IEnumerable<string> models, CancellationToken cancellationToken = default)
```

Comment:
Just to refresh my memory, what was the service reasoning for returning a non-pageable models array for the "create" operation, vs returning a pageable models array for "get" operation?

Reply:
We never asked. I posed the question to Brian Crawford this morning.

My guess is that in listing models, you'd potentially expect quite a few, making paging necessary. Also they use CosmosDB under the covers, which gives them paging for free.

However, model creation probably won't have that many models in the payload, so paging really shouldn't be necessary. Also, they probably can't rely on CosmosDB's paging to save them here.